### PR TITLE
Exception fixes for Python 3.10

### PR DESCRIPTION
**Pull request by lukasstarinsky**
Fixed exception errors that caused the program not to run properly.

### CODE REVIEW

**Comment by samuelmajtan** [comment]
nice work sir, keep going sir

**Comment by SimonZaloudek** [comment]
by ma zaujimalo ktory mental tam commitol pixelSize na 12,5 😄

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Running
```
Comment (janjanech) [inline README.md]: Please, remove README file, as it is not important in this PR

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+pyqt5
```
Comment (janjanech) [inline requirements.txt]: requirements file is already in repository, incorrectly named doc/dependencies.pip. Thank you for sugestion, I will rename the file, but remove yours please.

File: umlfri2/application/drawingarea/selection.py [hunk 3]
```diff
@@ -1,5 +1,7 @@
-from collections import Iterable
-
+try:
+    from collections.abc import Iterable
+except ImportError:
```
Comment (janjanech) [inline umlfri2/application/drawingarea/selection.py]: Fallback for the old import is not needed as all supported versions of Python 3 contains abc module.

File: umlfri2/plugin/communication/executor.py [hunk 4]
```diff
@@ -2,7 +2,11 @@
 import traceback
 import sys
 from base64 import b64encode
-from collections import Iterable
+
+try:
+    from collections.abc import Iterable
+except ImportError:
```
Comment (janjanech) [inline umlfri2/plugin/communication/executor.py]: Fallback for the old import is not needed as all supported versions of Python 3 contains abc module.

File: umlfri2/qtgui/mainwindow/tabs/startpageframe.py [hunk 5]
```diff
@@ -4,7 +4,6 @@
 
 from .startpageframeaction import StartPageFrameAction
 
-
```
Comment (janjanech) [inline umlfri2/qtgui/mainwindow/tabs/startpageframe.py]: PEP 8 requires two blank lines between top level definitions

**Review by janjanech — CHANGES_REQUESTED** [review]
Thank you for the contribution, but I need you to process some changes to be able to approve this PR.

**Comment by janjanech** [comment]
@SimonZaloudek that would be me as I am the sole author for now :wink: But I really have no idea how it got there

**Pull request merged**